server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java

```diff
@@ -873,7 +873,15 @@ void becomeCandidate(String method) {
             joinAccumulator = joinHelper.new CandidateJoinAccumulator();
 
             peerFinder.activate(coordinationState.get().getLastAcceptedState().nodes());
-            clusterFormationFailureHelper.start();
+
+            // Only start the formation helper if NOT in shutdown.
+            // The shutdown process will disconnect the node from the cluster, and start
+            // rejoining processes, the failure of which we do not want to log about.
+            if (lifecycle.initializedOrStarted()) {
+                clusterFormationFailureHelper.start();
+            } else {
+                logger.info("This node has left the cluster due to shutdown. It is unlikely to successfully rejoin.");
```

Comment:
I do not think it makes any sense to attempt to join the node while it is already shutting down.
I also do not think we need to log anything about it.

Reply:
Re my comment in the other thread, we appear to be logging quite a bit about election/joining, currently. Do you think we should instead be making a larger change to not activate the election/join logic during shutdown?